expression/typeinferer.go

```diff
@@ -82,6 +83,9 @@ func (v *typeInferrer) Leave(in ast.Node) (out ast.Node, ok bool) {
 		types.SetBinChsClnFlag(&x.Type)
 	case *ast.ParamMarkerExpr:
 		types.DefaultTypeForValue(x.GetValue(), x.GetType())
+		if v.useCache() {
```

Comment:
how about change it to `if v.sc.UseCache`, since function `useCache()` is so simple ?

Reply:
Thank you for the suggestion. I've changed the codes as follows:

`-      if v.useCache() {`
`+      if v.sc.UseCache {`
`           v.handleParamExpr(x)`
`       }`